middleware/markdown/summary/render.go

```diff
@@ -0,0 +1,143 @@
+package summary
+
+import (
+	"bytes"
+
+	"github.com/russross/blackfriday"
+)
+
+// Ensure we implement the Blackfriday Markdown Renderer interface
+var _ blackfriday.Renderer = (*Renderer)(nil)
+
+type Renderer struct{}
+
+// Blocklevel callbacks
+
+// BlockCode is the code tag callback.
+func (r Renderer) BlockCode(out *bytes.Buffer, text []byte, land string) {}
+
+// BlockQuote is teh quote tag callback.
+func (r Renderer) BlockQuote(out *bytes.Buffer, text []byte) {}
+
+// BlockHtml is the HTML tag callback.
+func (r Renderer) BlockHtml(out *bytes.Buffer, text []byte) {}
+
+// Header is the header tag callback.
```

Comment:
My bad, along with "teh" a few lines above -- all my code. I'm willing to fix later...